packages/twenty-server/src/engine/metadata-modules/relation-metadata/relation-metadata.service.ts

```diff
@@ -317,4 +318,42 @@ export class RelationMetadataService extends TypeOrmQueryService<RelationMetadat
 
     return deletedRelationMetadata;
   }
+
+  async getManyRelationMetadataByFieldMetadataIdsForDataloader(
```

Comment:
```suggestion
  async findManyRelationMetadataByFieldMetadataIds(
```
What do you think of that ?
This can be used outside DataLoader context too